internal/api/v1beta1connect/frontier_service_impl.go

```diff
@@ -0,0 +1,672 @@
+package v1beta1connect
+
+import (
+	"context"
+
+	"github.com/bufbuild/connect-go"
+	frontierv1beta1 "github.com/raystack/frontier/proto/v1beta1"
+)
+
+func (h *ConnectHandler) ListUsers(context.Context, *connect.Request[frontierv1beta1.ListUsersRequest]) (*connect.Response[frontierv1beta1.ListUsersResponse], error) {
+	return nil, connect.NewError(connect.CodeUnimplemented, nil)
+}
```

Comment:
We will move these functions to separate files as we implement them individually.

Reply:
Example implementation